ietf/meeting/migrations/0005_iabminutes.py

```diff
@@ -0,0 +1,959 @@
+# Copyright The IETF Trust 2023, All Rights Reserved
+
+import datetime
+import zoneinfo
+
+from django.db import migrations
+
+
+def nametimes_by_year():
+    with open("data_for_0005","r") as datafile:
+        return(eval(datafile.read())) # Consider making the dump json instead.
+
+def forward(apps, schema_editor):
+    Document = apps.get_model("doc", "Document")
+    DocAlias = apps.get_model("doc", "DocAlias")
+    Meeting = apps.get_model("meeting", "Meeting")
+    Schedule = apps.get_model("meeting", "Schedule")
+    Session = apps.get_model("meeting", "Session")
+    SchedulingEvent = apps.get_model("meeting", "SchedulingEvent")
+    SchedTimeSessAssignment = apps.get_model("meeting", "SchedTimeSessAssignment")
+    TimeSlot = apps.get_model("meeting", "TimeSlot")
+
+    ntby = nametimes_by_year()
+    for year in ntby.keys():
```

Comment:
Minor nit: probably better to iterate over a dict with `for year in ntby` without the `.keys()`. (Does the same thing, but does not have to set up a dictionary view.)